core/unix/src/TUnixSystem.cxx

```diff
@@ -4964,9 +4964,9 @@ static void GetDarwinProcInfo(ProcInfo_t *procinfo)
       mach_port_t object_name;
       vm_address_t address;
       vm_region_top_info_data_t info;
-      vm_size_t vsize, vprvt, rsize, size;
+      vm_size_t /*vsize,*/ vprvt, rsize, size;
```

Comment:
If the variable is not used below (only assigned), maybe we can completely get rid of it instead of commenting it out -so that the code is more readable.

Reply:
Thanks for the review! But the doxygen documentation contains a comment about vsize, what should I do with it?

Reply:
(see the suggestion above)